cosmos/reapchain_221230.json

```diff
@@ -0,0 +1,51 @@
+{
+  "rpc": "https://rpc.reapchain.org",
+  "rest": "https://lcd.reapchain.org",
+  "chainId": "reapchain_221230-1",
+  "chainName": "Reapchain Mainnet",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/reapchain/chain.png",
+  "stakeCurrency": {
+    "coinDenom": "REAP",
+    "coinMinimalDenom": "areap",
+    "coinDecimals": 18,
+    "coinGeckoId": "reapchain",
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/reapchain/areap.png"
+  },
+  "walletUrl": "https://wallet.keplr.app/chains/reapchain",
+  "walletUrlForStaking": "https://wallet.keplr.app/chains/reapchain",
```

Comment:
@helloreap Could you remove `walletUrl` and `walletUrlForStaking` field?
It is invalid url.